md-dao/src/main/java/com/wipro/ats/bdre/md/dao/ProcessDAO.java

```diff
@@ -280,6 +279,31 @@ public Process returnProcess(String processCode) {
 
     }
 
+
+
+
+    public List<Process> returnProcesses(String processCode) {
+
+        Session session = sessionFactory.openSession();
+        session.beginTransaction();
+        Process parentProcess=null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "parentProcess" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)